src/client/components/Dropdown/DropDown.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './DropDown.css';
+
+export function DropDown({ title }) {
```

Comment:
You should pass in an array of strings named `items` which you then render as a `<option>{value}</option>`.
If you are unclear about how to do that, contact me.